bin/container.sh

```diff
@@ -129,14 +125,13 @@ stop_container() {
   [ "${cid}" != "" ] && docker stop ${cid}
 }
 
-
 remove_container() {
   [ ${1} ] && container=${1}
 
   if [ ${container} ]; then
     stop_container ${container} >/dev/null 2>&1
     docker rm -fv ${container} >/dev/null 2>&1
-  else
+  elif [ "$OS" == "OSX" ]; then
     docker-machine rm -f ${project} >/dev/null 2>&1
```

Comment:
Question here: Would we ever want to remove the entire machine, since that will remove any cached images in that machine? Is it good enough to just remove the container?